static/js/stream_popover.js

```diff
@@ -262,6 +263,17 @@ function build_starred_messages_popover(e) {
 
 }
 
+function build_move_topic_to_stream_popover(e, message_id, stream_id, topic_name) {
+    const streams = stream_data.subscribed_subs().filter(s => s.stream_id !== stream_id);
```

Comment:
Let's call this variable `available_streams`.  

I'll note that we currently include private streams in this list.

Reply:
Done. Added a TODO for keyboard navigation of the dropdown.

Reply:
Do we want to exclude private streams?

Reply:
Well, I think there's a product question about whether it's correct to limit it to just public streams.  We should make a decision on that and adjust accordingly.  Initially we did it just to manage complexity.